### PR TITLE
remove duplication from package.yml

### PR DESCRIPTION
**Pull request by turboMaCk**
Hi :wave: 

I was working on the distribution of for several elm tools things using this project for nix os and I noticed there is a duplication in `package.yaml`.

### CODE REVIEW

File: package.yaml [hunk 1]
```diff
@@ -36,7 +36,6 @@ library:
   - async == 2.1.1.1
   - binary == 0.8.5.1
   - bytestring == 0.10.8.2
```
Comment (turboMaCk) [inline package.yaml]: :point_up:

File: package.yaml [hunk 2]
```diff
@@ -36,7 +36,6 @@ library:
...
-  - bytestring == 0.10.8.2
```
Comment (turboMaCk) [inline package.yaml]: duplicate

**Comment by stoeffel** [comment]
thanks!

**Comment by stoeffel** [comment]
@turboMaCk I published this with elmi-to-json@0.19.4.

**Pull request merged**